src/components/order/OrderedItemDetailInformation/OrderedItemDetailInformation.tsx

```diff
@@ -0,0 +1,51 @@
+import type { OrderData } from '../../../types';
+import { priceFormatter } from '../../../utils/formatter';
+import * as S from './OrderedItemDetailInformation.styles';
+
+type OrderedItemDetailInformationProps = OrderData;
+
+export const OrderedItemDetailInformation = ({
+  ...orderedItem
+}: OrderedItemDetailInformationProps) => {
```

Comment:
구조분해 할당을 해야하는 이유가 있나요...?

```suggestion
export const OrderedItemDetailInformation = (orderedItem: OrderedItemDetailInformationProps) => {
```

Reply:
없습니다😅 수정하였습니다!